docs/src/main/asciidoc/podman.adoc

```diff
@@ -8,73 +8,37 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 :topics: podman,devops,tooling
 include::_attributes.adoc[]
 
-https://podman.io/[Podman] is a daemonless and rootless container engine for developing, managing, and running OCI Containers on your Linux system or other OS.
-If you're using Podman with Quarkus, some one-off setup is needed, but once it's done, you can take advantage of all the Quarkus features.
+https://podman.io/[Podman] is a daemonless and rootless container engine for developing, managing, and running OCI Containers on Linux, Windows and OSX
 
-== Installing Podman
+Podman's approach varies dependent on the operating system you are using and also varied over time dpeendent on version of Podman. Thus albeit Podman comes with a cli based installer and available in various custom package installers (like brew or Linux package mangers) our default recommendation is to use the official Podman Desktop installer. This will ensure you get the best Developer Joy experience and support for your operating system.
 
-=== macOS
+See https://podman-desktop.io/downloads/ for the latest version of Podman Desktop or pick the version that suits your operating system from the list below:
 
-Containers are really Linux.
-As such, Linux containers cannot run natively on macOS or Windows.
-Therefore, the containers must run in a Linux virtual machine (VM), and a Podman client interacts with that VM.
-A native hypervisor subsystem and virtualization software is used to run the Linux VM on the OS, and then containers are run within this VM.
-In Podman, this is known as the Podman machine, and it is built into the tool.
+- https://podman-desktop.io/linux/[Linux]
+- https://podman-desktop.io/macos/[MacOS]
+- https://podman-desktop.io/windows/[Windows]
```

Comment:
As well as these links, should we also include the copy-pastable brew/winget/flatpak commands? They're pretty prominent on https://podman-desktop.io/downloads/, and mean a lot less clicks for people.